src/proc/StreamHandler.ts

```diff
@@ -11,7 +11,7 @@ export class StreamHandler {
      * Create a private constructor to prevent multiple instances
      */
     private constructor() {
-        this.currentSelectedSource = {};
+        this.currentSelectedSource = new MediaStream();
```

Comment:
Does this make sense? Defaulting the currentSelectedSource to a new media stream? Wouldn't null be better suited here?

Reply:
<img width="458" alt="image" src="https://user-images.githubusercontent.com/70775490/206309342-9f408410-da41-4ad0-b7ff-bc46efd0793d.png">
Doeskt work like that

Reply:
should we maybe disable strict null checks? Because this is stupid

Reply:
undefined could work as well

Reply:
<img width="467" alt="image" src="https://user-images.githubusercontent.com/70775490/206310624-8e3b06fd-22b3-4c09-acf9-8f1368272d37.png">
nope